js/Button/Repeat-Button.md

```diff
@@ -1,7 +1,7 @@
 ---
 layout: post
-title: Repeat-Button | Syncfusion
-description: repeat button
+title: Repeat-Button in JavaScript Button | Syncfusion
```

Comment:
don't use repeated keyword in single sentence
,
Repeat-Button in JavaScript | Syncfusion - change it.

Reply:
resolved